flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/LeaderChangeClusterComponentsTest.java

```diff
@@ -141,6 +142,16 @@ public void testReelectionOfJobMaster() throws Exception {
 		assertThat(jobResult.isSuccess(), is(true));
 	}
 
+	@Test
+	public void testTaskManagerRegisterReelectionOfResourceManager() throws Exception {
+
+		assertTrue(miniCluster.requestTaskManagerInfo().get().size() == NUM_TMS);
```

Comment:
I would suggest to use `assertThat` and Hamcrest matchers for testing because they give better failure messages.

Reply:
Ok, that sounds good.